lib/paylocity_web_service/authentication.rb

```diff
@@ -1,11 +1,12 @@
 require 'json'
 require 'base64'
+require 'digest/md5'
```

Comment:
It seems this is never used. do we need to use it somewhere?

Reply:
nice catch!

I have removed this line based on your review.

https://github.com/helloworld1812/paylocity-ruby-gem/commit/0430ce3cb59bb1834552da7cd5775f31c9365f64